ArcGISRuntimeSDKQt_QMLSamples/Features/FeatureLayer_DictionaryRenderer/FeatureLayer_DictionaryRenderer.qml

```diff
@@ -75,7 +67,10 @@ Rectangle {
                     // Create a dictionary renderer and apply to the layer
                     var renderer = ArcGISRuntimeEnvironment.createObject(
                                 "DictionaryRenderer",
-                                { dictionarySymbolStyle: dictionarySymbolStyle });
+                                { dictionarySymbolStyle:
+                                    //! [Create Dictionary Symbol Style QML]
+                                    DictionarySymbolStyle.createFromFile(dataPath + "/styles/arcade_style/mil2525d.stylx") });
```

Comment:
might be good if `});` were on a newline not included in the snippet. Otherwise that snippet will read a bit odd

Reply:
Fixed.